src/app/cluster/cluster-details/component.ts

```diff
@@ -343,6 +343,10 @@ export class ClusterDetailsComponent implements OnInit, OnDestroy {
     return MemberUtils.hasPermission(this._user, this._currentGroupConfig, 'rbac', Permission.View);
   }
 
+  isOPAEnabled(): boolean {
+    return !!this.cluster.spec.opaIntegration && !!this.cluster.spec.opaIntegration.enabled;
```

Comment:
No need for double negation of a non-optional boolean: `this.cluster.spec.opaIntegration.enabled`